### PR TITLE
Fix to the warning, using string sum instead

### PR DESCRIPTION
**Pull request by Vismayak**

### CODE REVIEW

**Review by longshuicy — CHANGES_REQUESTED** [review]
A few changes needed:
1. add changelog entry
2. keep the execution output
3. update the zip if you updated anything in the notebook

Also it would be help that you can pin-point which line did you change?

**Review by longshuicy — CHANGES_REQUESTED** [review]
- [x] Could you fix the conflict 
- [x] Code works well. But when I unzip I noticed there are additional macos files
image

- [ ] In the zip file; the notebook shouldn't contain any output cells. The output cell is only for notebooks under content/notebooks (for displaying on the documentation site)

**Comment by navarroc** [comment]
Checked the output against the previous and everything looks good. If you can address the issues @longshuicy identified, then I think it'll be ready to merge.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to the INCORE documents generated by Sphinx package will be
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased] -
```
[the remark this answers is not in the document]
Comment (Vismayak) [reply CHANGELOG.md]: All done, sorry for the stupid mistakes Chen! I am gonna get the changelog tatoo soon

**Comment by Vismayak** [comment]
@navarroc @longshuicy I think it is ready now to be merged :)

**Comment by Vismayak** [comment]
@navarroc removed 👍

**Comment by navarroc** [comment]
Sorry to nitpick, but can you remove the hidden .DS_Store file in the Galveston zip file? Thank you!

**Comment by navarroc** [comment]
Can you remove the __MACOSX folder in the zips?

**Review by Rashmil-1999 — APPROVED** [review]

**Review by longshuicy — APPROVED** [review]
zipfiles look good now

**Pull request merged**